util/db/dbutil.go

```diff
@@ -87,7 +87,11 @@ func MakeAccessor(dbfilename string, readOnly bool, inMemory bool) (Accessor, er
 // see https://www.sqlite.org/pragma.html#pragma_secure_delete
 // It is not read-only and not in-memory (otherwise, erasability doesn't matter)
 func MakeErasableAccessor(dbfilename string) (Accessor, error) {
-	return makeAccessorImpl(dbfilename, false, false, []string{"_secure_delete=on"})
+	return makeErasableAccessor(dbfilename, false)
+}
+
+func makeErasableAccessor(dbfilename string, readOnly bool) (Accessor, error) {
```

Comment:
I guess this function does not need `readOnly` argument, since `_secure_delete` does not make sense for reader

Reply:
It depend on the case - see new erasablepair

Reply:
I discovered our readOnly argument is not enforced by the driver, and mostly only controls whether `_txlock=immediate` is set in the URI. So I wanted to ensure that both of the pair of DB conns used the same _secure_delete flag when being opened, just in case.